templates/dashboard/index.html.twig

```diff
@@ -6,18 +6,18 @@
 
 {% block body %}
                 <div class="row">
-                    <div class="col-12 col-xl-6 mb-4 mb-lg-4">
+                    <div class="col-12 col-xxl-6 mb-4 mb-lg-4">
```

Comment:
please revert

Reply:
I'll look at it.

Reply:
i disagree.

Reply:
There's nothing to agree or disagree, it's just a fact which I tested. :-)
Below you can see the `col-xl-6` setting which wraps hostnames and reporters.

![image](https://github.com/antedebaas/DMARC-SMTPTLS-Reports/assets/44540/f4c38285-ce62-497c-8fbb-73394dd3b7fb)

Reply:
That’s because you widened the menu causing the grid to be misaligned

Reply:
This was necessary because originally the menu items wrapped and were misaligned.
Choose your poison ... :-)